pkg/ebpf/net_events.go

```diff
@@ -203,13 +203,12 @@ func (t *Tracee) processNetEvents(ctx gocontext.Context) {
 					continue
 				}
 
-				if t.config.Debug {
+				if t.eventsToTrace[NetPacket] {
```

Comment:
add verification of the packet interface with the asked interface in `-trace` flag